test/library/packages/Sort/performance/sort-performance-explorer.chpl

```diff
@@ -1,5 +1,5 @@
 // This tester allows exploration of sort performance
```

Comment:
Do you think we should retire our other [sort perf tests](https://chapel-lang.org/perf/chapcs/?graphs=linearithmicsortson224bytesofshuffleddata,quadraticsortson212bytesofshuffleddata)?

Reply:
No, sort-performance-explorer isn't actually a proper perf test (but one day it will be a real boy).